### DISCUSSION TITLE
Failed to load language config: unknown field `language-server`

### DISCUSSION THREAD

**Discussion by andradei** [helix-editor/helix#7439] [Troubleshooting]
After adding the GDScript LSP config to my `languages.toml` I got this error. I also don't see any reference to `language-server` (plural) in the documentation despite it being in multiple places in the Wiki:

![image](https://github.com/helix-editor/helix/assets/2653546/4fecd6d2-4cdd-4c69-868c-3c3a0443226f)

**Comment by pascalkuthe** [comment]
There was a breaking change to to config on master. If you are still using 23.5 or older than that config doesn't work yet. Consult the stable docs.

**Reply by andradei** [reply]
Yeah, I'm on `23.05`. The Wiki seems to have a mix of bleeding edge and stable configuration that aren't compatible with each other, but that's the nature of an editable Wiki. Looks like a lot of people is running `master` 😄 

Thank you for the explanation. Looking forward to the next version.
